pxtsim/runtime.ts

```diff
@@ -306,8 +306,8 @@ namespace pxsim {
             return id;
         }
 
-        unregisterLiveObject(object: RefObject) {
-            U.assert(object.refcnt == 0, "ref count is not 0");
+        unregisterLiveObject(object: RefObject, keepAlive?: boolean) {
+            if (!keepAlive) U.assert(object.refcnt == 0, "ref count is not 0");
```

Comment:
?

Reply:
Equivalent of noLeakTracking in v0. Sorry it's slightly unrelated change.
https://github.com/Microsoft/pxt/blob/v0/pxtsim/langsupport.ts#L38